### PR TITLE
Fix slack v3 docs

### PR DESCRIPTION
**Pull request by YuvHayun**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [XSUP-28210](https://jira-hq.paloaltonetworks.local/browse/XSUP-28210)

## Description
Documentation and metadata improvements.

### CODE REVIEW

File: Packs/Slack/Integrations/SlackV3/README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ to learn about configuring SlackV3 using the app manifest.
     | `incidentType` | Type of incidents created in Slack. | False |
     | `allow_incidents` | Allow external users to create incidents via direct messages. | False |
     | `proxy` | Use system proxy settings. | False |
-    | `unsecure` | Trust any certificate (not secure). | False |
+    | `unsecure` | Trust any certificate (not secure). Make sure to mark this param if you want SlackBlockBuilder script to send response back to the incident context. | False |
```
Comment (ShirleyDenkberg) [inline Packs/Slack/Integrations/SlackV3/README.md:28]:
```suggestion
    | `unsecure` | Trust any certificate (not secure). Make sure to mark this parameter if you want the SlackBlockBuilder script to send a response back to the incident context. | False |
```

File: Packs/Slack/Integrations/SlackV3/README.md [hunk 2]
```diff
@@ -727,4 +727,11 @@ Retrieves replies to specific messages, regardless of whether it's from a public
...
-There is no context output for this command.
\ No newline at end of file
+There is no context output for this command.
+```
+
+## Troubleshooting
+---
+**Issue**: The survey sent from SlackBlockBuilder is sent to Slack and submitted successfully, but the response does not show up in context data in Cortex XSOAR.
+
+**Troubleshooting**: The most likely cause is that there is no API key entered into the Slack v3 integration instance settings, or the API key was not created by default admin. Ensure an API key created by a default admin user is entered into the Slack v3 integration instance settings. Also, make sure to mark the **Trust any certificate (not secure)** integration param.
```
Comment (ShirleyDenkberg) [inline Packs/Slack/Integrations/SlackV3/README.md:737]:
```suggestion
**Resolution**: The most likely cause is that there is no API key entered into the Slack v3 integration instance settings, or the API key was not created by the default admin user. Ensure that an API key created by a default admin user is entered into the Slack v3 integration instance settings. Also, make sure to mark the **Trust any certificate (not secure)** integration parameter.
```

File: Packs/Slack/Integrations/SlackV3/SlackV3.yml [hunk 3]
```diff
@@ -96,6 +96,7 @@ configuration:
   section: Connect
   required: false
   advanced: true
+  additionalinfo: Trust any certificate (not secure). Make sure to mark this param if you want SlackBlockBuilder script to send response back to the incident context.
```
Comment (ShirleyDenkberg) [inline Packs/Slack/Integrations/SlackV3/SlackV3.yml:99]:
```suggestion
  additionalinfo: Trust any certificate (not secure). Make sure to mark this parameter if you want the SlackBlockBuilder script to send a response back to the incident context.
```

File: Packs/Slack/Scripts/SlackBlockBuilder/README.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-SlackBlockBuilder will format a given Slack block into a format readable by the SlackV3 integration. The script will also send the block to the given destination.
+SlackBlockBuilder will format a given Slack block into a format readable by the SlackV3 integration. The script will also send the block to the given destination. Make sure to mark **Trust any certificate** and fill the **XSOAR API Key** integration params if you wish to get the response to the incident context.
```
Comment (ShirleyDenkberg) [inline Packs/Slack/Scripts/SlackBlockBuilder/README.md:1]:
```suggestion
SlackBlockBuilder will format a given Slack block into a format readable by the SlackV3 integration. The script will also send the block to the given destination. Make sure to mark **Trust any certificate** and fill the **XSOAR API Key** integration parameters if you want to get a response to the incident context.
```

File: Packs/Slack/Scripts/SlackBlockBuilder/README.md [hunk 5]
```diff
@@ -120,7 +120,7 @@ Verify your Slack blocks payload is valid. Try simplifying the payload. Test wit
 ---
 **Issue**: The survey is sent to Slack and submitted successfully, but the response does not show up in context data in Cortex XSOAR.
 
-**Troubleshooting**: The most likely cause is that there is no API key entered into the Slack v3 integration instance settings, or the API key was not created by default admin. Ensure an API key created by a default admin user is entered into the Slack v3 integration instance settings.
+**Troubleshooting**: The most likely cause is that there is no API key entered into the Slack v3 integration instance settings, or the API key was not created by default admin. Ensure an API key created by a default admin user is entered into the Slack v3 integration instance settings. Also, make sure to mark the **Trust any certificate (not secure)** integration param.
```
Comment (ShirleyDenkberg) [inline Packs/Slack/Scripts/SlackBlockBuilder/README.md:123]:
```suggestion
**Troubleshooting**: The most likely cause is that there is no API key entered into the Slack v3 integration instance settings, or the API key was not created by default admin. Ensure an API key created by a default admin user is entered into the Slack v3 integration instance settings. Also, make sure to mark the **Trust any certificate (not secure)** integration parameter.
```

File: Packs/Slack/Scripts/SlackBlockBuilder/SlackBlockBuilder.yml [hunk 6]
```diff
@@ -7,7 +7,7 @@ type: python
 tags:
 - slack
 enabled: true
-comment: SlackBlockBuilder will format a given Slack block into a format readable by the SlackV3 integration. The script will also send the block to the given destination.
+comment: SlackBlockBuilder will format a given Slack block into a format readable by the SlackV3 integration. The script will also send the block to the given destination. Make sure to mark Trust any certificate and fill the XSOAR API Key integration params if you wish to get the response to the incident context.
```
Comment (ShirleyDenkberg) [inline Packs/Slack/Scripts/SlackBlockBuilder/SlackBlockBuilder.yml:10]:
```suggestion
comment: SlackBlockBuilder will format a given Slack block into a format readable by the SlackV3 integration. The script will also send the block to the given destination. Make sure to mark **Trust any certificate** and fill the **XSOAR API Key integration** parameters if you want to get a response to the incident context.
```

**Comment by ShirleyDenkberg** [comment]
@JudahSchwartz Doc review completed.

**Review by JudahSchwartz — APPROVED** [review]

**Pull request merged**